02.extensions/zooseo/01.basics/06.faq/docs.md

```diff
@@ -4,4 +4,10 @@ taxonomy:
     category: docs
 ---
 
-* **Is ZOOseo compatible with other Joomla! SEO extensions?** <br /> Yes, ZOOseo extension works with any Joomla! SEO extensions. They are complementary and work together.
\ No newline at end of file
+* **Why doesn't metadata load on the Item page?** - Please make sure that ZOOseo plugin is enabled and setup, and "ZOOseo Metadata Item" layout is configured.
```

Comment:
Don't use the (") carachter, use instead bold, cursive or code (**, *, `). If ZOOseo is enabled, already is set, so change the text to: "Please make sure that ZOOseo plugin is enabled and the *ZOOseo Metadata Item* layout is populated."